mobile/lib/services/hash.service.dart

```diff
@@ -65,7 +65,13 @@ class HashService {
       if (hashes[i] != null) {
         continue;
       }
-      final file = await assets[i].local!.originFile;
+
+      File? file;
+
+      try {
+        file = await assets[i].local!.originFile;
+      } catch (_) {}
```

Comment:
This should log the error with a stack trace. I know there's a warning below, but it's generic and doesn't say what the actual error was.

Reply:
Done